src/app/client/src/app/modules/workspace/components/published/published.component.spec.ts

```diff
@@ -18,12 +18,14 @@ import { TelemetryModule } from '@sunbird/telemetry';
 import { NgInviewModule } from 'angular-inport';
 import { SuiModule } from 'ng2-semantic-ui';
 import { CoreModule } from '@sunbird/core';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```

Comment:
Here also include all the dependecies provides and modules them this is not required

Reply:
Here also include the missing components in the mocking module